sat/cli/bootprep/input/image.py

```diff
@@ -250,6 +270,10 @@ def begin_image_create(self):
 
         LOGGER.info(f'Launching IMS job to create image')
 
+        ims_base = self.ims_base
```

Comment:
I do believe this is an uncommon use case, and I believe it's actually a pre-existing bug. It would be handled by [CRAYSAT-1529](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1529). I am going to add your comment here to that Jira.